### ISSUE TITLE
Issues with 0.16.1

### ISSUE THREAD

**Issue by laicasaane** [favoyang/unity-addressable-importer#87]
#### Bug category

- [x] Importer: issues related to the importer behaviours or UI.
- [ ] ODIN: issues related to the Odin integration.
- [ ] Documentation
- [ ] Others

#### Steps to reproduce
- Upgrade to 0.16.1 from 0.16.0

#### What is expected?
- Everything works fine as before.

#### What is actually happening?
1. It throws a warning after upgrading
```
[AddressableImporter] import settings file not found.
Please go to Assets/AddressableAssetsData folder, right click in the project window and choose 'Create > Addressables > Import Settings'.
—
at void UnityEngine.Debug.LogWarningFormat(string format, params object[] args)
void AddressableImporter.OnPostprocessAllAssets(string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromAssetPaths) in ./Library/PackageCache/com.littlebigfun.addressable-importer@0.16.1/Editor/AddressableImporter.cs:58
```
2. And if we try to recreate the importer settings, it throws an exception
```
NullReferenceException: Object reference not set to an instance of an object
—
System.NullReferenceException: Object reference not set to an instance of an object
void AddressableImportSettings.CreateAsset() in ./Library/PackageCache/com.littlebigfun.addressable-importer@0.16.1/Editor/AddressableImportSettings.cs:86
```

#### Other relevant information

- OS (Win/Mac/Linux/Others): Windows
- Unity version: 2022.2.5f1
- Addressables version: 1.21.2
- Unity Addressable Importer version: 0.16.1

**Comment by favoyang** [comment]
That's weird.

0.16.1 from 0.16.0 only applies the commit below. It doesn't seem related to your issue.
https://github.com/favoyang/unity-addressable-importer/commit/5ebbcd8a48232ad3282c1f5589d8aaa5e5dffc46

I try to update 0.16.0 to 0.16.1 and get no problem. Also, it works by deleting both setting files and recreating them from scratch.

```
AddressableImportSettings
AddressableImportSettingsList
```

void AddressableImportSettings.CreateAsset() in ./Library/PackageCache/com.littlebigfun.addressable-importer@0.16.1/Editor/AddressableImportSettings.cs:86

```
        if (!AddressableImportSettingsList.Instance.SettingList.Contains(settings))
```

Could you place a few Debug.Log to check which element is null?

```
AddressableImportSettingsList
AddressableImportSettingsList.Instance
AddressableImportSettingsList.Instance.SettingList
settings
```

**Comment by laicasaane** [comment]
> 0.16.1 from 0.16.0 only applies the commit below. It doesn't seem related to your issue.

Yeah. I also inspected the said commit but can't understand anywhere that could possibly cause this issue.

I'll try again later by deleting the Library folder.
